taxonomy/models.py

```diff
@@ -0,0 +1,31 @@
+"""Models for lore taxonomy application"""
+
+from django.db import models
+
+
+class Vocabulary(models.Model):
+    repository = models.ForeignKey("Repository")
+    name = models.TextField()
+    description = models.TextField()
+    required = models.BooleanField()
+    type = models.CharField(max_length=1, choices=(
```

Comment:
It's not technically a reserved word (https://docs.python.org/2/reference/lexical_analysis.html#keywords) but I see your point. Should I call it _type and specify 'type' as the database column name?